src/plugins/integrations/AWSIntegration.ts

```diff
@@ -101,7 +102,6 @@ class AWSIntegration implements Integration {
               tags: {
                 [SpanTags.SPAN_TYPE]: SpanTypes.AWS_SQS,
                 [AwsSDKTags.REQUEST_NAME]: operationName,
-                [AwsSQSTags.QUEUE_NAME]: queueName,
                 [SpanTags.OPERATION_TYPE]: operationType ? operationType : 'READ',
```

Comment:
we might use empty operation type instead of `READ` if the operation is not known

Reply:
Done